src/Common/src/System/Threading/Tasks/TaskValue.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Threading.Tasks
+{
+    /// <summary>Value type discrimated union for a TResult and a <see cref="Task{TResult}"/>.</summary>
+    /// <typeparam name="TResult">The type of the result.</typeparam>
+    internal struct TaskValue<TResult>
+    {
+        /// <summary>The task. this will be non-null iff the operation didn't complete successfully synchronously.</summary>
```

Comment:
If "iff" usage indicates "if and only if", not a typo.



On Sat, Feb 28, 2015 at 5:32 AM, Dmitry Turin <notifications@github.com>
wrote:

> In src/Common/src/System/Threading/Tasks/TaskValue.cs
> <https://github.com/dotnet/corefx/pull/1048#discussion_r25556186>:
>
> > @@ -0,0 +1,118 @@
> > +// Copyright (c) Microsoft. All rights reserved.
> > +// Licensed under the MIT license. See LICENSE file in the project root for full license information.
> > +
> > +using System.Diagnostics;
> > +using System.Runtime.CompilerServices;
> > +
> > +namespace System.Threading.Tasks
> > +{
> > +    /// <summary>Value type discrimated union for a TResult and a <see cref="Task{TResult}"/>.</summary>
> > +    /// <typeparam name="TResult">The type of the result.</typeparam>
> > +    internal struct TaskValue<TResult>
> > +    {
> > +        /// <summary>The task. this will be non-null iff the operation didn't complete successfully synchronously.</summary>
>
> typo: "if"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotnet/corefx/pull/1048/files#r25556186>.
>